tools/clang/unittests/HLSL/ExecutionTest.cpp

```diff
@@ -939,7 +956,7 @@ class ExecutionTest {
       CComPtr<ID3D12Device10> pDevice10;
       // Copy resDesc0 to resDesc1 zeroing anything new
       D3D12_RESOURCE_DESC1 resDesc1 = {0};
-      memcpy(&resDesc1, &resDesc, sizeof(resDesc));
+      CopyDesc0ToDesc1(resDesc1, resDesc) {
```

Comment:
Looks like there was a typo here, should be `CopyDesc0ToDesc1(resDesc1, resDesc);` but has an open curly brace :(